.github/workflows/build_and_run.yml

```
@@ -165,8 +165,14 @@ jobs:
           &$script_path
           echo "OCL_ICD_FILENAMES=$env:CONDA_PREFIX\Library\lib\intelocl64.dll" >> $env:GITHUB_ENV
 
-      - name: Run benchmarks
-        run: dpbench -i ${{env.WORKLOADS}} run -r2 --no-print-results || exit 1
+      - name: Run benchmarks fp64
+        run: dpbench -i ${{env.WORKLOADS}} run -r2 --no-print-results --precision=double || exit 1
```

Let's add `precision: ["single", "double"]` to matrix instead of running tests sequentially
```suggestion
        run: dpbench -i ${{env.WORKLOADS}} run -r2 --no-print-results --precision={{matrix.precision}} || exit 1
```